doc/source/whatsnew/v0.17.0.txt

```diff
@@ -136,21 +136,22 @@ numeric types when there were no values convertible to a numeric type.  For exam
      0  a
      1  b
 
-returns the original DataFrame with no conversion. This change alters
-this behavior so that
+  returns the original DataFrame with no conversion. This change alters
+  this behavior so that
 
   .. ipython:: python
 
     pd.DataFrame({'s': ['a','b']})
     df.convert_objects(numeric=True, coerce=True)
 
-converts all non-number-like strings to ``NaN``.
+    converts all non-number-like strings to ``NaN``.
```

Comment:
this one is not yet correct, should be the same level as the 'this behaviour so that' above the code block